GroceryStoreRESTfullCRUD/src/main/java/com/store/components/entity/Cheque.java

```diff
@@ -0,0 +1,80 @@
+package com.store.components.entity;
+
+public class Cheque {
+	private int id;
+	private int idReserve;
+	private int idOrder;
+	private int idCustomer;
+	private int idSale;
+	private float amount;
+	private String orderDate;
+
+	public Cheque() {
+	}
+
+	public Cheque(int id, int idReserve, int idOrder, int idCustomer, int idSale, float amount, String orderDate) {
```

Comment:
is this constructor necessary? Do you somewhere use it? Same for other entities.